tox.ini

```diff
@@ -38,7 +38,8 @@ commands =
 
 [testenv:typing]
 basepython = {env:PYTHON3_PATH:python3}
+whitelist_externals=/bin/bash
 deps =
      -r{toxinidir}/requirements_test.txt
 commands =
-         mypy --ignore-missing-imports --follow-imports=skip homeassistant
+         /bin/bash -c 'mypy --ignore-missing-imports --follow-imports=skip homeassistant/*.py'
```

Comment:
The mypy docs recommend using `silent` instead of `skip`.
http://mypy.readthedocs.io/en/latest/command_line.html#follow-imports

Reply:
When I run with silent there's one additional error in bootstrap on line 301.

Reply:
Done